### PR TITLE
Added imagePullSecrets to aws-ssm-agent-installer chart

### PR DESCRIPTION
**Pull request by nikhilbagdi**
Added imagePullSecrets to aws-ssm-agent-installer chart.

### CODE REVIEW

File: charts/aws-ssm-agent-installer/values.yaml [hunk 1]
```diff
@@ -10,3 +10,5 @@ tolerations:
     operator: Exists
   - effect: NoSchedule
     operator: Exists
+imagePullSecrets:
+  - cred
```
Comment (prakarsh-dt) [inline charts/aws-ssm-agent-installer/values.yaml:14]: Can you please remove this default value or comment it out?

Comment (nikhilbagdi) [reply charts/aws-ssm-agent-installer/values.yaml]: Yes, done.

File: charts/aws-ssm-agent-installer/values.yaml [hunk 2]
```diff
@@ -10,3 +10,5 @@ tolerations:
     operator: Exists
   - effect: NoSchedule
     operator: Exists
+imagePullSecrets:
```
Comment (prakarsh-dt) [inline charts/aws-ssm-agent-installer/values.yaml:13]: imagePullSecrets: [] else it will give error

Comment (nikhilbagdi) [reply charts/aws-ssm-agent-installer/values.yaml]: Yes, done.

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**